tests/DependencyInjection/AwsExtensionTest.php

```diff
@@ -12,46 +12,20 @@
 class AwsExtensionTest extends TestCase
 {
     /**
-     * @var ContainerInterface
+     * @var array
      */
-    protected $container;
+    protected $args;
 
     public function setUp()
     {
-        $kernel = new AppKernel('test', true);
-        $kernel->boot();
-
-        $this->container = $kernel->getContainer();
-    }
-
-    /**
-     * @test
-     */
-    public function sdk_should_be_accessible_as_a_service()
-    {
-        $this->assertTrue($this->container->has('aws_sdk'));
-    }
-
-    /**
-     * @test
-     * @dataProvider serviceProvider
-     *
-     * @param string $webServiceName
-     * @param string $containerServiceName
-     * @param string $clientClassName
-     */
-    public function all_web_services_in_sdk_manifest_should_be_accessible_as_container_services(
-        $webServiceName,
-        $containerServiceName,
-        $clientClassName
-    ) {
-        $this->assertTrue(
-            $this->container->has($containerServiceName)
-        );
-
-        $service = $this->container->get($containerServiceName);
```

Comment:
As with the config test below, couldn't this test just be modified to fetch a public service into which a few SDK clients were injected?